auth-tokens-filter/src/test/java/com/palantir/tokens/auth/http/BearerTokenLoggingContextFilterTests.java

```diff
@@ -95,15 +104,49 @@ public void testBearerTokenLogging_userIdAppearsInMetaDataContext() {
     }
 
     @Test
-    public void testBearerTokenLogging_userIdAppearsInrequestLog() {
+    public void testBearerTokenLogging_sessionIdAppearsInMetaDataContext() {
```

Comment:
could you inline these tests with testBearerTokenLogging_userIdAppearsInMetaDataContext ? there's a lot of duplicate code to check basically the same thing.